src/core/server/base.js

```diff
@@ -74,7 +75,10 @@ export default function(routes, createStore) {
 
       const store = createStore();
 
-      store.dispatch({type: 'SET_JWT', payload: {token: cookie.load('jwt_api_auth_token')}});
+      const token = cookie.load(config.get('cookieName'));
```

Comment:
If you land this before my config patch I'll fix these up to work with the "new world order".